spec/system/galleries/index_spec.rb

```diff
@@ -20,7 +20,7 @@ def expect_gallery_of(name, size, tags=[])
     gallery_row = find('tr') { |x| x.has_selector?('.gallery-name a', exact_text: name) }
     within(gallery_row) do
       expect(page).to have_selector('.gallery-icon-count', exact_text: size.to_s)
-      tag_box = page.all('.tag-box').first
+      tag_box = page.first('.tag-box')
```

Comment:
Based on the tests it looks like this was depending on `.all(...).first` to get `nil` if it doesn't exist. I wonder if there's a good way to be explicit about that?

Reply:
Is there more than one `.tag-box` on the page in this gallery_row? (I guess not)

Maybe we can just do `page.all('.tag-box .tag-item-link')` directly or something, let me dig into this quickly

Reply:
I think that'll still give you an array?

Reply:
for future reference, we could do `.first(, minimum: 0)` instead (but this should be clearer)